docs/integrations/destinations/clickhouse.md

```diff
@@ -1,12 +1,18 @@
 # ClickHouse
 
+:::warning
+The Clickhouse connector is outdated and does not use Destination v2 (typing and deduplication). 
+As a result, it may not work well with large datasets or could have performance issues. 
+If this connector is critical to your project, please consider upvoting and adding comments to the [discussion]([url](https://github.com/airbytehq/airbyte/discussions/35339)) about transitioning Clickhouse to the Destination v2 format. 
```

Comment:
This is [the most upvoted open discussion](https://github.com/airbytehq/airbyte/discussions?discussions_q=is%3Aopen+sort%3Atop+created%3A%3E%3D2023-12-10) of the past year. It feels strange to ask for this kind of engagement here. Unless we're hoping to get more out of the discussion, maybe we can just say "See the GitHub discussion to learn more" or something like that?

Reply:
Agree, changes the sentence to Subscribe to the discussion this way people will receive updates about the topic.